packages/CodingStandard/tests/Fixer/Property/ArrayPropertyDefaultValueFixerTest.php

```diff
@@ -0,0 +1,57 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Fixer\Property;
+
+use PhpCsFixer\Fixer\FixerInterface;
+use PhpCsFixer\Test\AbstractFixerTestCase;
+use PHPUnit\Framework\Constraint\IsIdentical;
+use PHPUnit\Framework\TestCase;
+
+// support for PHPUnit 6 missing :(
+class_alias(TestCase::class, 'PHPUnit_Framework_TestCase');
+class_alias(IsIdentical::class, 'PHPUnit_Framework_Constraint_IsIdentical');
+
+final class ArrayPropertyDefaultValueFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @dataProvider provideFixCases()
+     */
+    public function testFix(string $expected, string $input)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    /**
+     * @return string[][]
+     */
+    public function provideFixCases(): array
+    {
+        return [
+            [
+                '<?php
+
+                class SomeClass
+                {
+                    /**
+                     * @var int[]
+                     */
+                    public $property = [];
```

Comment:
I think there is a another fixer for that.

But...

How to include it in the Fixer? Not sure how to detect, if is property on next line or not.